magda-search-api/src/main/scala/au/csiro/data61/magda/search/elasticsearch/Queries.scala

```diff
@@ -22,7 +22,9 @@ object Queries {
   def publisherQuery(strategy: SearchStrategy)(publisher: FilterValue[String]) = {
     handleFilterValue(publisher, (publisherString: String) =>
       strategy match {
-        case SearchStrategy.MatchAll => matchQuery("publisher.name.keyword_lowercase", publisherString)
+        case SearchStrategy.MatchAll => multiMatchQuery(publisherString)
+          .fields("publisher.acronym", "publisher.name.keyword_lowercase")
+          .minimumShouldMatch("-50%")
         case SearchStrategy.MatchPart =>
           multiMatchQuery(publisherString)
             .fields("publisher.name", "publisher.name.english")
```

Comment:
should probably put the acronym here too :)

Reply:
Thanks a lot!
Have corrected this~